MetaMask/metamask-extension
# UX: Multichain: Restore Flask support link and text

## Explanation

In reviewing the old account menu, the Flask build uses a custom link and link text for support.  I'm adding that same customization for the new Multichain global menu.

## Manual Testing Steps

1.  `MULTICHAIN=1 yarn start --build-type flask`
2. Open the global menu
3. See different text and link for Flask

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
